pkg/check/privilege.go

```diff
@@ -121,10 +121,18 @@ func verifyPrivileges(result *Result, grants []string, expectedGrants []string)
 		firstGrant = firstGrant + " 'secret'"
 	}
 
+	// Aurora has some privilege failing parsing
+	firstGrant = strings.Replace(firstGrant, "LOAD FROM S3", "", 1)
+	firstGrant = strings.ReplaceAll(firstGrant, ", ,", ",")
```

Comment:
Can we remove this line and only let L128 handle the same logic?

Reply:
one replacing of this kind is not enough, because
`..., LOAD FROM S3, SELECT INTO S3,...` -> `..., , ,...` -> `..., ,...` ⬅️three adjacent comma will only trigger once.